src/manifest.json

```diff
@@ -11,7 +11,6 @@
   "permissions": [
     "storage",
     "unlimitedStorage",
-    "videoCapture",
```

Comment:
Why `videoCapture` permission is removed?

Reply:
It is not a valid entry for permissions

Firefox: 
![firefox](https://github.com/aeternity/superhero-wallet/assets/49337371/9051944b-74ef-4183-bee7-0bbe0296f481)
Chrome:
![chrome](https://github.com/aeternity/superhero-wallet/assets/49337371/269d98fa-37d6-4dbb-914a-9c8e54303cd5)